src/NuGet.Core/NuGet.Configuration/Settings/SettingsFile.cs

```diff
@@ -39,6 +39,13 @@ internal sealed class SettingsFile
         /// <remarks>Machine wide settings files cannot be eddited.</remarks>
         internal bool IsMachineWide { get; }
 
+        /// <summary>
+        /// Determines if the settings file is read only. 
```

Comment:
Which component is responsible for physical file permissions?

```suggestion
        /// Determines if the settings file is considered read-only by NuGet. 
```

Reply:
> Which component is responsible for physical file permissions?

None, we don't mess with permissions when it comes to configuration files. 
We trust that the process permissions are good enough.